index.js

```diff
@@ -1 +1,13 @@
 // code your solution here
+
+const superbowlWin = record => {
+  const yearWon = record.find(findWin)
+  if (typeof yearWon === 'undefined'){
+    return yearWon
+  } else {
+    return yearWon.year
+  }
```

Comment:
`undefined` is falsy. You could simply use a ternary here to do the exact same thing: 
```suggestion
  return yearWon ?  yearWon.year : yearWon
```
OR, even better, just do this:
```suggestion
  if (yearWon) { return yearWon.year } 
```